src/main.tsx

```diff
@@ -1,12 +1,12 @@
 import { StrictMode } from 'react';
 import { createRoot } from 'react-dom/client';
 
-import App from './App.tsx';
+import Router from './router/router.tsx';
```

Comment:
```suggestion
import Router from './router/router';
```

일반적으로 확장자를 생략해요~
이유는 나중에 파일을 .tsx에서 .ts로 바꾸거나, .js로 바꿔도 import 문을 수정할 필요가 없어서 리팩터링시 용이하고 가독성이 더 좋습니다.
그리고 vite또는 webpack등 번들러 설정으로 확장자를 자동 해석하도록 되어있어서 생략해도 무방합니다~